web/src/api/storage/types/config.ts

```diff
@@ -0,0 +1,369 @@
+/* eslint-disable */
+/**
+ * This file was automatically generated by json-schema-to-typescript.
```

Comment:
We should decide on a tool to generate those files, but that's out of the scope of this PR.

Reply:
Sure, BTW this types are fetched from the storage-config-ui branch (https://github.com/agama-project/agama/blob/storage-config-ui/web/src/api/storage/types/config.ts)